### DISCUSSION TITLE
Finding coordinates on map

### DISCUSSION THREAD

**Discussion by ericvoshall** [Hypfer/Valetudo#1695] [Q&A / Support] [answered]
Hi, I am trying to configure an automation in Home Assistant to have my RoboRock S5 go to a specific location on the map. I can't seem to figure out how to determine coordinates for the map. Any help would be appreciated!

**Comment by Hypfer** [comment, marked as answer]
https://github.com/Hypfer/Valetudo/discussions/1688
